cloudify_kubernetes/tasks.py

```diff
@@ -158,6 +158,60 @@ def _do_resource_status_check(resource_kind, response):
         else:
             ctx.logger.debug('status {0}'.format(status))
 
+    elif resource_kind == 'Deployment':
+        conditions = response['status']['conditions']
+        if conditions:
```

Comment:
I would probably do:
```python
if isinstance(conditions, list):
  for condition in conditions:
```

Reply:
Yeah I think this is much better , will update it